queue_services/entity-filer/src/entity_filer/filing_processors/consent_continuation_out.py

```diff
@@ -0,0 +1,41 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the court order filing."""
+from contextlib import suppress
+from datetime import datetime
+from typing import Dict
+
+import dpath
+from dateutil.relativedelta import relativedelta
+from legal_api.models import Business, Filing
+
+from entity_filer.filing_meta import FilingMeta
+
+
+def process(business: Business, cco_filing: Filing, filing: Dict, filing_meta: FilingMeta):
+    """Render the court order filing into the business model objects."""
+    # update court order, if any is present
+    with suppress(IndexError, KeyError, TypeError):
+        court_order_json = dpath.util.get(filing, '/consentContinuationOut/courtOrder')
+        cco_filing.court_order_file_number = court_order_json.get('fileNumber')
+        cco_filing.court_order_effect_of_order = court_order_json.get('effectOfOrder')
+        cco_filing.order_details = filing['consentContinuationOut'].get('details')
+
+    business.cco_expiry_date = datetime.now() + relativedelta(months=6)
```

Comment:
We want to make sure the time portion of the expiry date uses the same time like we do in other scenarios.  Specifically, the date will be need to be saved as midnight UTC e.g. _2023-08-17T08:00:00.000000+00:00._

The FE will expect the following format when it retrieves the expiry date in the `filings.meta_data`.  

``` json
{
  "legalFilings": [
    "consentContinuationOut"
  ],
  "applicationDate": "2023-04-17T17:11:00.372000+00:00",
  "consentContinuationOut": {
    "expiry": "2023-08-17T08:00:00.000000+00:00"
  }
}

```

The restoration expiry date is a bit different in that it gets the expiry date from the filing json but it may still be something you can use as a reference.  https://github.com/bcgov/lear/blob/ac42907c4c42f6999c9858209e30f5ef7da7825e/queue_services/entity-filer/src/entity_filer/filing_processors/restoration.py#L56-L55